dbcrossbarlib/src/drivers/bigquery/write_remote_data.rs

```diff
@@ -50,14 +50,14 @@ pub(crate) async fn write_remote_data_helper(
 
     // Decide if we need to use a temp table.
     let (use_temp, initial_table_name) = if !schema.bigquery_can_import_from_csv()? {
-        let initial_table_name = dest.table_name.clone();
+        let initial_table_name = dest.table_name.temporary_table_name();
```

Comment:
There were some bugs in the BigQuery import driver with the handling of temporary and final table names. I think these canceled out during one of our internal data loads, because we got a correctly decoded table with the correct name anyway. But I had to fix them

Probably we minimize the risk of future bugs like this via some combination of:

1. Better encapsulating the verbose code that shells out to `bq load`.
2. Maybe _always_ using a temp table for imports, even if there are no `ARRAY` columns.